internal/mq/mqimpl/rocksmq/server/rocksmq_impl.go

```diff
@@ -787,30 +798,45 @@ func (rmq *rocksmq) Consume(topicName string, groupName string, n int) ([]Consum
 		dataKey = path.Join(topicName, strconv.FormatInt(currentID, 10))
 	}
 	iter.Seek([]byte(dataKey))
+	iterPropersity.Seek([]byte(dataKey))
+
 	consumerMessage := make([]ConsumerMessage, 0, n)
 	offset := 0
+
 	for ; iter.Valid() && offset < n; iter.Next() {
 		key := iter.Key()
 		val := iter.Value()
 		strKey := string(key.Data())
 		key.Free()
-		offset++
+		properties := make(map[string]string)
+		var propertiesValue []byte
+
 		msgID, err := strconv.ParseInt(strKey[len(topicName)+1:], 10, 64)
 		if err != nil {
 			val.Free()
 			return nil, err
 		}
-		askedProperties := path.Join(common.PropertiesKey, topicName, strconv.FormatInt(msgID, 10))
-		opts := gorocksdb.NewDefaultReadOptions()
-		defer opts.Destroy()
-		propertiesValue, err := rmq.store.GetBytes(opts, []byte(askedProperties))
-		if err != nil {
-			return nil, err
+		offset++
+
+		if iterPropersity.Valid() && string(iterPropersity.Key().Data()) == string(iter.Key().Data()) {
+			// the key of properties is the same with the key of payload
+			// to prevent mix message with or without propersity column family
+			propertiesValue = iterPropersity.Value().Data()
+			iterPropersity.Next()
+		} else {
+			// between 2.2.0 and 2.3.0, the key of Payload is topic/properties/msgid/Payload
```

Comment:
let's ignore the data for 2.3.0 beta.
Just make sure property empty is ok for 2.3